### ISSUE TITLE
regression: ZendXml component missing from 2.4.12 tarball

### ISSUE THREAD

**Issue by glensc** [zendframework/zendframework#7770]
https://packages.zendframework.com/releases/ZendFramework-2.4.12/ZendFramework-2.4.12.tgz

Only in ZendFramework-2.4.11/library: ZendXml

changelog does not mention it's being removed, so must be packaging mistake:

```
## 2.4.12 (2017-06-19)

- Fix signature issue with AbstractContainer::offsetGet
```

---

```
mkdir test
cd test/
wget https://packages.zendframework.com/releases/ZendFramework-2.4.11/ZendFramework-2.4.11.tgz
wget https://packages.zendframework.com/releases/ZendFramework-2.4.12/ZendFramework-2.4.12.tgz

tar xf ZendFramework-2.4.11.tgz 
tar xf ZendFramework-2.4.12.tgz 

diff -qr ZendFramework-2.4.1[12] | grep Only
Only in ZendFramework-2.4.11/library: ZendXml
```

**Comment by glensc** [comment]
in fact [2.2.6 (2014-03-06)](
https://github.com/zendframework/zendframework/blob/release-2.4.12/CHANGELOG.md#226-2014-03-06) lists ZendXml being added under [**SECURITY**](https://github.com/zendframework/zendframework/blob/release-2.4.12/CHANGELOG.md#security-updates-13):

> ZF2014-01: Potential XXE/XEE attacks using PHP functions: simplexml_load_*, DOMDocument::loadXML, and xml_parse. A new component, ZendXml, was introduced to mitigate XML eXternal Entity and XML Entity Expansion vectors that are present in older versions of libxml2 and/or PHP. Zend\Json\Json::fromXml() and Zend\XmlRpc's Response and Fault classes were potentially vulnerable to these attacks. If you use either of these components, we recommend upgrading immediately.

/cc @weierophinney

**Comment by froschdesign** [comment]
No package version for version 2.4.10, 2.4.11 and 2.4.12 here on GitHub. This also means `ZendXml` is missing.

See: https://github.com/zendframework/zendframework/releases

**Comment by glensc** [comment]
@froschdesign don't understand what you're saying. no package version for what? zendxml, zendframework? screenshot to explain you better?

zendxml does not exist in 2.4 tree, i think its latest version was just pulled in from
https://github.com/zendframework/ZendXml/releases

**Comment by glensc** [comment]
ps: when making 2.4.13 release to fix bad packaging. please include this fix (at least): https://github.com/zendframework/zendframework/issues/7704#issuecomment-314909650

**Comment by froschdesign** [comment]
@glensc 
See the release page:

![zf](https://user-images.githubusercontent.com/103362/28154605-0b04c106-67ab-11e7-9c15-1e0bc30b13b3.png)

The first two downloads are package versions. These downloads include `ZendXml`.

**Comment by weierophinney** [comment]
Fixed with version 2.4.13, released a short bit ago.
